tensorflow/compiler/xla/service/gpu/cusolver_context.cc

```diff
@@ -17,6 +17,22 @@ limitations under the License.
 
 #include "tensorflow/compiler/xla/util.h"
 
+#if !defined(TENSORFLOW_USE_ROCM)
+
+using gpuStream_t = cudaStream_t;
+
+#else
+
+#include "rocm/include/hip/hip_complex.h"
+
+using gpuStream_t = hipStream_t;
```

Comment:
To minimize the difference, WDYT about an even more local style, e.g.

```
using gpuStream_t = 
#ifdef ROCM
hipStream_t
#else
cudaStream_t
#endif
```

Also ideally these would be all gathered in some common file in stream_executor, so we could just use common defines